OpenRA.Mods.Common/Widgets/Logic/Ingame/WorldTooltipLogic.cs

```diff
@@ -19,7 +19,7 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 	public class WorldTooltipLogic : ChromeLogic
 	{
 		[TranslationReference]
-		static readonly string UnrevealedTerrain = "unrevealed-terrain";
+		static readonly string UnrevealedTerrain = "tooltip-unrevealed-terrain";
```

Comment:
could the translation be cashed? It's called every frame

Reply:
I'd like to keep this PR focused on translation IDs and the FTL file structure. Let's address these kind of issues in separate PRs. Putting scope aside those are also more urgent and this PR might take a while to complete.

Reply:
Adressed in #20269